web-console/src/dialogs/status-dialog/status-dialog.scss

```diff
@@ -31,14 +31,14 @@ $side-bar-width: 120px;
 
     .bp3-dialog-footer {
       position: relative;
-      display:inline;
+      display: inline;
```

Comment:
Maybe we can add a linter as part of CI?

Reply:
I realized my issue...my prettier/file watcher was only configured to autoformat .tsx files and didn't include .scss files.